### DISCUSSION TITLE
Camera access

### DISCUSSION THREAD

**Discussion by chp77** [pedroSG94/RootEncoder#1325] [Q&A]
Can we lock the camera and prevent other apps from accessing it while streaming?

**Comment by pedroSG94** [comment]
Hello,

No, you can't do it. It is managed by the system. In my case using a pixel 6a (Android 14) and Oneplus 7T(Android 12) I tested it using RTPService example.
Just start stream and move app to background, then open other app like native app camera and you can see in the player side that video is stopped but you will have the video again when you resume the my app to foreground.
Since this happens with 2 different devices I assume that it is the way it should works in all devices

**Comment by chp77** [comment]
Hi, I'm using the rtmpService.
When I started the live streaming, opened the other camera apps, and went back to my live Stream app, the camera was not able to capture the view correctly (stopped at the last view) but had audio.
Can I know how to ensure that the camera view is captured correctly after I switch to the live stream app?

**Comment by pedroSG94** [comment]
Hello,

If you tested with RTP service example, try to check that preview is working after come back to the RTP service activity. Also, check that you can see video packets in logcat in that moment. If only the player is freeze, try to re start stream in the player and let me know if this is the problem to find a way to solve it.

Just in case, in my tests I put the activity in background and then come again to the activity using the mobile menu where you can see apps in background (square button in the navigation bar). 
If you are doing tests using different steps let me know it. Also, I will need device model and Android version to reproduce your case.
